client/stores/working_form.js

```diff
@@ -165,7 +166,13 @@ export const useWorkingFormStore = defineStore("working_form", {
       const field = newFields.splice(oldIndex, 1)[0]
       newFields.splice(newIndex, 0, field)
       this.content.properties = newFields
-    }
+    },
+    setSubmissionHash(key, hash) {
```

Comment:
We shouldn't store the submission hash here. this is not dependent on currnet form + this is to store form data while we work on it